Podfile

```diff
@@ -3,8 +3,8 @@ platform :osx, '10.12'
 target 'alt-tab-macos' do
   use_frameworks!
   pod 'LetsMove', :git => 'https://github.com/lwouis/LetsMove.git', :commit => '7abf4daed1a25218f2b52f2dfd190aee5a50071c'
-  pod 'Sparkle', :podspec => 'https://raw.githubusercontent.com/lwouis/Sparkle/master/Sparkle.podspec'
+  pod 'Sparkle', '1.24.0'
```

Comment:
I see that you had to upgrade Sparkle for Universal compatibility. Unfortunately, Sparkle is missing a fix I made on my fork of Sparkle back in the day.

I submitted a [PR](https://github.com/sparkle-project/Sparkle/pull/1938) to them today, but that's on their `master` which is v1.27.0 I think. This version changes the webview and other things, so not sure if it works with AltTab setup.

In the meanwhile, I updated the my fork to host a fixed v1.24.0. This `Podfile` should now include `pod 'Sparkle', :podspec => 'https://raw.githubusercontent.com/lwouis/Sparkle/fix-iframe-popup-1.24/Sparkle.podspec'`

Reply:
Cool, thanks, there was a new comment on your PR 7 days ago https://github.com/sparkle-project/Sparkle/pull/1938#issuecomment-912801855

If nothing else comes up for the time being, I would then switch to your version.

Reply:
My PR, if merged, would only fix `master`. Then they would need to release v1.28. Then we would need to move to that, and I think it has breaking change, so we would need to work on that.

I don't know the Sparkle project enough to be sure, but I suspect they don't support fixes on old releases. Something like v1.24.1 releases today, which is what we want ideally. I think it's simpler to use the branch on my fork which is exactly that: v1.24 + the fix = v1.24.1. That way we get Universal Build support without the popups bug, and without having to deal with the breaking changes introduces after v1.24

Reply:
@lwouis which version should I take now? The MR is closed, shall we use the beta v2 version of Sparkle?

Reply:
They [closed by MR indeed](https://github.com/sparkle-project/Sparkle/pull/1938), and mentioned that the fixes may be included in future releases.

In September, I prepared a fork of Sparkle 1.24.0, which is AppleSilicon compatible, with the fixes needed by AltTab cherry picked. It is probably the best approach for now: https://github.com/lwouis/Sparkle/blob/fix-iframe-popup-1.24/Sparkle.podspec

Reply:
Is it fixed in the beta version 2?